### ISSUE TITLE
Prevent shown_time bar from extending beyond the bounds of time_slider div

### ISSUE THREAD

**Issue by cchan987** [nengo/nengo-gui#86]

**Comment by tcstewar** [comment]
Can you describe this bug?  I thought I had it checking for this.  that's what this code is for:

```javascript
if (new_time > self.last_time - self.shown_time) {
                new_time = self.last_time - self.shown_time;
            }
            if (new_time < self.last_time - self.kept_time) {
                new_time = self.last_time - self.kept_time;
            }
```

**Comment by cchan987** [comment]
Here's a screenshot:
![image](https://cloud.githubusercontent.com/assets/10402911/7070169/35e1579e-deab-11e4-9b8d-1388cc5663e9.png)

First extend the shown_time bar so that it fills the whole time_slider area, then when you continue to drag outwards, it will go off the right hand side

**Comment by tcstewar** [comment]
Ah, I see -- the left edge is being checked, but not the right edge.

**Comment by tcstewar** [comment]
Fixed in #155
